draft-ietf-quic-http.md

```diff
@@ -696,18 +696,19 @@ requests or pushes were accepted prior to the connection shutdown.  Endpoints
 SHOULD abruptly terminate any requests or pushes that have identifiers greater
 than or equal to the smallest identifier sent in a GOAWAY frame.
 
-Endpoints MUST NOT initiate new requests or pushes on the connection with an
-identifier greater than or equal to the smallest identifier received in a GOAWAY
-frame.  Clients MAY establish a new connection to send additional requests.
+Endpoints MUST NOT initiate new requests or promise new pushes on the connection
+after receipt of a GOAWAY from the peer.  Clients MAY establish a new connection
```

Comment:
Maybe change GOAWAY to "GOAWAY frame"? I think we consistently use the latter.

Reply:
Thanks -- I found a few other places where just "GOAWAY" was used, and fixed them as well.